seq2science/rules/configuration_generic.smk

```diff
@@ -107,27 +113,35 @@ for key, value in config.items():
 
 # read the samples.tsv file as all text, drop comment lines
 samples = pd.read_csv(config["samples"], sep='\t', dtype='str', comment='#')
-samples.columns = samples.columns.str.strip()
```

Comment:
Is it really necessary to crash the pipeline for an extra white space?

Reply:
Yeah maybe you are right, it's a matter of preference I guess.